.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react-native"]
```

Comment:
Not sure if we need this one as well, what's the purpose of it?

Reply:
Babel uses it to compile the code for the tests.

https://github.com/facebook/react-native/tree/master/babel-preset

Reply:
Ah, okay, great!